pennylane/operation.py

```diff
@@ -121,6 +121,7 @@
     ~CVOperation
     ~Channel
     ~Tensor
+    ~StatePrep
```

Comment:
And adding it to the `inheritance-diagram` directly below. I tried to get the diagram to be automatic, but sphinx was troublesome.